### PR TITLE
Archer v2 - edited adding an api endpoint

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
edited building a url with special endpoint.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ArcherRSA/Integrations/ArcherV2/README.md [hunk 1]
```diff
@@ -23,6 +24,8 @@ The RSA Archer GRC Platform provides a common foundation for managing policies,
 | fields_to_fetch | List of fields from the application to gets into the incident | False |
 
 4. Click **Test** to validate the URLs, token, and connection.
+
+P.S : the api_endpoint parameter should be used only if you have another api endpoint.
```
Comment (yaakovi) [inline Packs/ArcherRSA/Integrations/ArcherV2/README.md:28]:
```suggestion
Note: the api_endpoint parameter should be used only if you have another API endpoint.
```

File: Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml [hunk 2]
```diff
@@ -1,3979 +0,0 @@
-commonfields:
```
Comment (yaakovi) [inline Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml:1]: revert

File: Packs/ArcherRSA/ReleaseNotes/1_1_22.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### RSA Archer v2
+- Updated the Docker image to: *demisto/python3:3.9.6.24019*.
+- fixed an issue where the *api_endpoint* paramter was not added properly for the url , after releasing the previous new version.
```
Comment (yaakovi) [inline Packs/ArcherRSA/ReleaseNotes/1_1_22.md:5]:
```suggestion
- fixed an issue where the *api_endpoint* parameter was not added properly to the URL.
```

**Review by yaakovi — CHANGES_REQUESTED** [review]

File: Packs/ArcherRSA/ReleaseNotes/1_1_22.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### RSA Archer v2
+- Updated the Docker image to: *demisto/python3:3.9.6.24019*.
+- fixed an issue where the *api_endpoint* parameter was not added properly to the URL.
```
Comment (yaakovi) [inline Packs/ArcherRSA/ReleaseNotes/1_1_22.md:5]: missed it the last time...
```suggestion
- Fixed an issue where the *api_endpoint* parameter was not added properly to the URL.
```

**Review by yaakovi — APPROVED** [review]

File: Packs/ArcherRSA/ReleaseNotes/1_1_22.md (hunk 4 above)
Comment (bakatzir) [reply Packs/ArcherRSA/ReleaseNotes/1_1_22.md:5]: Use the display name `Advanced: API Endpoint`
customers can't see(and do not mind) the `name` key.

**Pull request merged**